src/breeze/components/breeze/sidebar_link.cr

```diff
@@ -1,9 +1,10 @@
-class Breeze::SidebarLink(T) < BreezeComponent
+class Breeze::SidebarLink < BreezeComponent
   needs context : HTTP::Server::Context
-  needs content : String
+  needs link_text : String
+  needs link_to : Lucky::Action.class | Lucky::RouteHelper
```

Comment:
I moved this here because if you want to pass a link that includes query or route params, you can't pass that in as a Generic. But calling this `link_to` method will throw a `Invalid call` error to a random Lucky Action in your app. 😖